usaspending_api/api_contracts/contracts/v2/recipient/state/fips.md

```diff
@@ -36,4 +36,5 @@ This endpoint returns a high-level overview of a specific state or territory, gi
     The year the median household income is based on.
 + `total_prime_amount`: 300200000000 (required, number)
 + `total_prime_awards`: 327721 (required, number)
++ `face_value_loan_guarantee`: 88888.0 (required, number)
 + `award_amount_per_capita`: 916023.08 (required, number)
```

Comment:
I think the mock will require count of award loans in the API response as well.

Reply:
I added the count